src/lib/Containers/Conversation.tsx

```diff
@@ -86,21 +83,9 @@ export class Conversation extends React.Component<RelayProps, any> {
   }
 
   render() {
-    const conversation = this.props.me.conversation
+    const conversation = this.props.viewer.me.conversation
     const partnerName = conversation.to.name
-    const artwork = conversation.artworks[0]
-    const temporaryTimestamp = "11:00AM"
-
-    // Ideally we will use a Relay fragment in the Message component, but for now this is good enough
-    const messageData = conversation.messages.edges.reverse().map(({ node }, index) => {
-      return {
-        senderName: this.isFromUser(node) ? conversation.from.name : partnerName,
-        key: index,
-        time: temporaryTimestamp,
-        body: node.snippet,
-      }
-    })
-
+    const messages = this.props.viewer.me.conversation.messages.edges.map(edge => edge.node)
```

Comment:
So for some reason, it's as if the fragments are being ignored, or something: all the `nodes` are just empty objects.